src/consensus/params.h

```diff
@@ -58,6 +56,10 @@ struct Params {
     int BIP65Height;
     /** Block height at which BIP66 becomes active */
     int BIP66Height;
+    /** Block height at which CSV becomes active */
+    int CSVHeight;
+    /** Block height at which Segwit becomes active */
```

Comment:
Is it helpful to continue referencing the BIP numbers in these comments? I would guess so.

Reply:
Sure. Comments added.

Reply:
Why not refer to the BIP numbers solely instead of alternating?

Reply:
the segwit deployment covered more than one bip